bsp/esp32_s3_korvo_1/README.md

```diff
@@ -0,0 +1,12 @@
+# BSP: ESP32-S3-KORVO-1
+
+[![Component Registry](https://components.espressif.com/components/espressif/esp32_s3_korvo_1/badge.svg)](https://components.espressif.com/components/espressif/esp32_s3_korvo_1)
+
+* [Hardware Reference](https://github.com/espressif/esp-skainet/blob/master/docs/en/hw-reference/esp32s3/user-guide-korvo-1.md)
+
+![image](pic.png)
```

Comment:
Please change to absolute path, so the image is correctly rendered in https://components.espressif.com/